src/lib/core/styles/tokens/field/_tokens.scss

```diff
@@ -8,136 +8,58 @@
 @use '../../utils';
 
 $tokens: (
-  text-color: utils.module-val(field, text-color, theme.variable(text-high)),
+  background: utils.module-val(field, background, transparent),
+  tonal-background: utils.module-val(field,tonal-background, theme.variable(surface-container-low)),
+  tonal-background-hover: utils.module-val(field, tonal-background-hover, theme.variable(surface-container-minimum)),
+  filled-background: utils.module-val(field, filled-background, theme.variable(surface-bright)),
 
-  // Spacing (label)
-  label-margin-inline: utils.module-val(field, label-margin-inline, 8px),
-  label-margin-block: utils.module-val(field, label-margin-block, 4px),
-  label-gap: utils.module-val(field, label-gap, 0.25em),
+  outline-style: utils.module-val(field, outline-style, solid),
+  outline-width: utils.module-val(field, outline-width, border.variable(thin)),
```

Comment:
Are we missing `outline-color`? The outline doesn't seem to override properly in dark theme.

Reply:
I removed the dedicated `outline-color` token and replaces it with just the `outline-low` global theme token. I *did* forget to apply it to the default theme, though, so that explains the lack of dark theme support.